elasticsearch/client/ingest.py

```diff
@@ -46,16 +46,16 @@ def delete_pipeline(self, id, params=None):
             'pipeline', id), params=params)
 
     @query_params('verbose')
-    def simulate(self, body, id=None, params=None):
+    def simulate(self, body, pipelineID=None, params=None):
```

Comment:
Usually in Python in order not to shadow builtins you can append an underscore: `id_`.
If you want to use this name, I think you should change it to `pipeline_id`.